lib/resources/kernel_module.rb

```diff
@@ -1,20 +1,62 @@
 # encoding: utf-8
 # author: Christoph Hartmann
 # author: Dominik Richter
+# author: Aaron Lippold
+# author: Adam Leff
 
 module Inspec::Resources
   class KernelModule < Inspec.resource(1)
     name 'kernel_module'
-    desc 'Use the kernel_module InSpec audit resource to test kernel modules on Linux platforms. These parameters are located under /lib/modules. Any submodule may be tested using this resource.'
+    desc 'Use the kernel_module InSpec audit resource to test kernel modules on
+    Linux platforms. These parameters are located under /lib/modules. Any submodule
+    may be tested using this resource.
+
+    The `kernel_module` resource can also verify if a kernel module is `blacklisted`
+    or if a module is disabled via a fake install using the `bin_true` or `bin_false`
+    method.'
+
     example "
-      describe kernel_module('bridge') do
-        it { should be_loaded }
-      end
+
+    describe kernel_module('video') do
+      it { should be_loaded }
+      it { should_not be_disabled }
```

Comment:
I'm 👎  on that. I'd like us to pick one (I'd prefer `disabled?`) and then use `should` or `should_not` accordingly.